client/api_crosschain.go

```diff
@@ -0,0 +1,122 @@
+package client
+
+import (
+	"context"
+	"cosmossdk.io/math"
+	sdkmath "cosmossdk.io/math"
+	gnfdSdkTypes "github.com/bnb-chain/greenfield/sdk/types"
+	bridgetypes "github.com/bnb-chain/greenfield/x/bridge/types"
+	storagetypes "github.com/bnb-chain/greenfield/x/storage/types"
+	crosschaintypes "github.com/cosmos/cosmos-sdk/x/crosschain/types"
+	oracletypes "github.com/cosmos/cosmos-sdk/x/oracle/types"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type CrossChain interface {
+	TransferOut(ctx context.Context, toAddress string, amount math.Int, txOption *gnfdSdkTypes.TxOption) (*sdk.TxResponse, error)
+
+	Claims(ctx context.Context, srcShainId, destChainId uint32, sequence uint64, timestamp uint64, payload []byte, voteAddrSet []uint64, aggSignature []byte, txOption *gnfdSdkTypes.TxOption) (*sdk.TxResponse, error)
+	GetChannelSendSequence(ctx context.Context, channelId uint32) (uint64, error)
+	GetChannelReceiveSequence(ctx context.Context, channelId uint32) (uint64, error)
+	GetInturnRelayer(ctx context.Context, req *oracletypes.QueryInturnRelayerRequest) (*oracletypes.QueryInturnRelayerResponse, error)
+	GetCrossChainPackage(ctx context.Context, channelId uint32, sequence uint64) ([]byte, error)
+
+	MirrorGroup(ctx context.Context, id sdkmath.Uint, txOption *gnfdSdkTypes.TxOption) (*sdk.TxResponse, error)
```

Comment:
How about using group_id to  make the interface clearer?

Reply:
The below MirrorBucket/MirrorObject is also the same.

Reply:
fixed